src/commands/build/wranglerjs/metadata.rs

```diff
@@ -0,0 +1,16 @@
+use serde::Serialize;
+
+#[derive(Serialize, Debug)]
+pub struct Bindings {
```

Comment:
We should make this generic and move out of webpack specific project type. Each binding type has different configuration values. This is what I was referring to in #273. We don't need to add all possible binding types, but make it possible add them in the future and make it possible to use the same binding type for both KV namespaces and wasm modules.

Reply:
i need to brush up on my rust type system, but i think maybe binding is a trait... #hottake. gonna write this into the design doc i think.

Reply:
just to update, @cwndrws clarified that yes that's exactly what he was saying.